apps/site/lib/site_web/views/helpers/time_helpers.ex

```diff
@@ -1,9 +1,29 @@
 defmodule SiteWeb.TimeHelpers do
+  @moduledoc """
+  Various helper functions for formatting dates/times.
+  """
   use Timex
 
-  @doc "Returns a string with the full month, day and year."
+  @type format_type :: atom()
+
+  @doc "Returns a string with the month, day and year, according to format type"
+  @spec format_date(DateTime.t() | Date.t(), format_type() | nil) :: String.t()
   @spec format_date(DateTime.t() | Date.t()) :: String.t()
-  def format_date(date) do
+  def format_date(date, format_type \\ nil)
+
+  def format_date(date, :event) do
```

Comment:
Hmmm. So I think if this is going to include logic pertaining to events, then it should be in a file more specific to that than this one.

However, I'm also wondering about a possibly more generalized approach that I've seen used in `SiteWeb.ViewHelpers.pretty_date()`:

```elixir
def pretty_date(date, format \\ "{Mshort} {D}") do
    if date == Util.service_date() do
      "today"
    else
      Timex.format!(date, format)
    end
  end
```

 What if we had a helper function where the arguments can dictate the format? Then you wouldn't need separate format_month or format_day, it could just be `format_event_date(date, "{Mshort}")` (for example)

Reply:
I can also see the value in the less general approach, where we have functions just to format dates for events, but I'd like to see those in the event view rather than this more general file. 
So, up to you.. There are really so so many ways to implement a thing. 😩

Reply:
Yeah, so true!!  I can see why event-specific logic should hang out in the event view.  And I ❤️ the pretty_date function, now that I know it exists.

I'm trying to think of a good way to extract _or_ extrapolate the relevant functions from the `cms_view` file, too...  The event view reaches out to the cms view for the `render_duration` function, which then reaches out to the time helpers file for the actual Timex calucaltions.  So hm... thinking through the best way to organize this all 
 in the events view without duplicating code

Reply:
You know what, I don't think it'd be a crazy messy to just create a `render_event_duration` func in the event view.  Imma do it 😁